pages_list.py

```
@@ -1,50 +1,57 @@
+            - A NoSuchElementException is caught and logged if no <a> elements are found on the page.
+            - The method uses self.cfg.geckodriver_path for specifying the path to the geckodriver executable.
+            - All browser operations are executed in headless mode.
+        """
         self.logger.debug('PagesList.ReadPagesList() started')
         # Настройка Firefox для работы в headless-режиме
         options = Options()
         options.add_argument("--headless")  # Включаем headless-режим
 
         try:
-            service = webdriver.ChromeService(executable_path="/usr/local/bin/geckodriver")
+            service = webdriver.ChromeService(executable_path=self.cfg.geckodriver_path)
```

Ruff (0.8.2)

`webdriver` may be undefined, or defined from star imports

(F405)